### PR TITLE
Added mParticle guides for Android and iOS

### PR DESCRIPTION
**Pull request by TheMarshal99**

### CODE REVIEW

**Comment by TheMarshal99** [comment]
@aeromusek

File: pages/getting-started/deep-link-routing.md [hunk 1]
```diff
@@ -738,9 +808,9 @@ let sessionParams = Branch.getInstance().getLatestReferringParams()
 {% endtabs %}
 {% endif %}
 
-{% if page.android %}
+{% if page.android or page.mparticle_android %}
 {% highlight java %}
-JSONObject sessionParams = Branch.getInstance().getLatestReferringParams();
+JSONObject sessionParams = Branch.get{% if page.mparticle_android %}Test{% endif %}Instance().getLatestReferringParams();
```
Comment (aeromusek) [inline pages/getting-started/deep-link-routing.md]: @TheMarshal99 why does mParticle use getTestInstance() here? Is that definitely correct?

File: pages/getting-started/deep-link-routing.md [hunk 2]
```diff
@@ -800,9 +870,9 @@ let firstParams = Branch.getInstance().getFirstReferringParams()
 {% endtabs %}
 {% endif %}
 
-{% if page.android %}
+{% if page.android or page.mparticle_android %}
 {% highlight java %}
-JSONObject installParams = Branch.getInstance().getFirstReferringParams();
+JSONObject installParams = Branch.get{% if page.mparticle_android %}Test{% endif %}Instance().getFirstReferringParams();
```
Comment (aeromusek) [inline pages/getting-started/deep-link-routing.md]: Same question as above ^

File: pages/getting-started/sdk-integration-guide.md [hunk 3]
```diff
@@ -24,6 +26,10 @@ sections:
 Before using the Branch SDK, you must first [sign up for an account](https://dashboard.branch.io){:target="_blank"} and complete the [onboarding process](https://start.branch.io/){:target="_blank"}.
 {% endprerequisite %}
 
+{% prerequisite %}
```
Comment (aeromusek) [inline pages/getting-started/sdk-integration-guide.md]: We should bury this inside an if statement. Right now it's showing up on the standard install instructions for all platforms

**Comment by aeromusek** [comment]
@TheMarshal99 added a couple notes, but otherwise I think we should just get this merged in. I'm going to reassess the presentation of these platforms soon anyway, so we can always move it somewhere else later!

File: pages/getting-started/deep-link-routing.md (hunk 1 above)
Comment (TheMarshal99) [inline pages/getting-started/deep-link-routing.md]: I think I meant to write "Auto" instance! >.<

**Comment by aeromusek** [comment]
Okayyyy let's do this 👍

**Pull request merged**